engine/apps/telegram/models/connectors/personal.py

```diff
@@ -172,3 +169,11 @@ def send_link_to_channel_message(self, alert_group: AlertGroup, notification_pol
                 )
             else:
                 raise e
+
+    @staticmethod
+    @ignore_reply_to_message_deleted
```

Comment:
Using the `@ignore_reply_to_message_deleted` decorator is the main change of this PR